### DISCUSSION TITLE
Bug in prisma generate

### DISCUSSION THREAD

**Discussion by victor-dm-black** [prisma/prisma#15366] [Q&A]
prisma@last_version
When I create record in my model

I get error `Unique constraint failed on the fields: (id)`
```ts
await prisma.botUsers.create({
		data: {
			telegramId: BigInt(1),
			telegramBotId: BigInt(3), userId: 1,
			role: 'Admin', botId: 9
		}
	});
```

```prisma
model BotUsers {
  id            Int      @id @default(autoincrement())
  userId        Int
  telegramId    BigInt
  telegramBotId BigInt
  role          RoleName @default(User)
  bot           Bots?    @relation(fields: [botId], references: [id])
  user          Users?   @relation(fields: [userId], references: [id])
  botId         Int
  inviterId     BigInt?
}
```
```ts
type BotUsersCreateInput {
  telegramId: BigInt
  telegramBotId: BigInt
  role?: RoleName
  bot?: BotsCreateNestedOneWithoutBotUsersInput
  user?: UsersCreateNestedOneWithoutBotUsersInput
  inviterId?: BigInt | Null
}
```

**Comment by austincrim** [comment]
Hey @victor-dm-black 👋,

Do you have existing data in the `botUsers` table that might conflict with the data you are inserting?

What database are you using?

**Reply by vladi-stevanovic** [reply]
👋 Hello @victor-dm-black did you perhaps have a chance to check Austin's questions? Let us know if this is still an open Discussion, or we'd love to know how you solved this!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
